Makefile

```diff
@@ -38,7 +38,12 @@ SED = sed
 
 .DEFAULT: $(TARGET)
 $(TARGET): $(SOURCES) Makefile $(GENERATED_FILES)
-	go build -ldflags "-X main.Version=$(VERSION_COMMIT)" -o $@ .
+	@build_flags="" ; \
+	if [ "$$RACE_DETECTION" = true ] ; then \
```

Comment:
@sboeuf The $$ is for using shell variables in a Makefile. A single $foo would cause make to look for a make variable foo, the use of $$foo would result in a literal "$foo" to be passed to the shell script which would then evaluate it correctly.